src/component/BooksRender.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RemoveBooks from './Removebook';
+
+const Books = ({ id, title, author }) => (
+  <div className="bookSite">
+    <div className="info">
+      <div className="left-section">
+        <ul>
+          <li className="genre books">Action</li>
+          <div>
+            <li className="booktitle books">{title}</li>
+          </div>
+          <li className="bookauthor books">{author}</li>
+        </ul>
+
+        <div className="book-actions">
+          <button className="book-btn" type="button">
+            Comments
+          </button>
+          <span>|</span>
+          <RemoveBooks id={id} />
+          <span>|</span>
+          <button className="book-btn" type="button">
+            Edit
+          </button>
+        </div>
+      </div>
```

Comment:
- [ ] I recommed following the Zeplin design the same colors and navbar and the sections since I can see a lot of differences between the Given design and your design so kindly make sure to change the Navbar to be the same and all the following sections displaying the books and the form 🔥💯🚀✨

- Your design
![image](https://user-images.githubusercontent.com/58553711/204062496-399670ce-f6ea-4956-b826-302920bb0b67.png)


- Zeplin design
![image](https://user-images.githubusercontent.com/58553711/204062470-88b57974-b12c-4574-9ee5-c9bc72489684.png)

Reply:
ok fixed